src/pages/index.js

```diff
@@ -1,65 +1,73 @@
-      link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
+  const api = new Api({
+    baseUrl: "https://around-api.en.tripleten-services.com/v1",
+    headers: {
+      authorization: "9b29ae94-bb6f-470b-bccd-5f1bdf13a16a",
+      "Content-Type": "application/json",
     },
+  });
 
-    {
-      name: "Lake Louise",
-      link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
-    },
+  let initialCardsInfo = NaN;
+
+  api.getInitialcards().then((items) => {
+    // handle the response
+    initialCardsInfo = items.reverse();
+    console.log(initialCardsInfo);
+    const cardsSection = new Section(
+      {
+        items,
+        renderer: (cardData) => {
+          console.log("---- oo ----", cardData._id);
+          addCard(cardData.name, cardData.link, cardTemplate, cardData._id);
+        },
+      },
+      galleryList
+    );
+    cardsSection.renderItems();
+  });
 
-    {
-      name: "Bald Mountains",
-      link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-    },
+  // try to incorporate PopupWithForm {problems with inputvalues}
+  const editAvatarPopup = new avatarPopup("#avatar-modal", chageAvatar);
+  const avatarPhoto = document.querySelector(".profile__avatar");
 
-    {
-      name: "Latemar",
-      link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-    },
+  function chageAvatar(inputValue) {
+    console.log(inputValue);
+    avatarPhoto.src = inputValue;
+    api.changeAvatar(inputValue);
+  }
 
-    {
-      name: "Vanoise National Park",
-      link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
-    },
+  avatarPhoto.addEventListener("click", () => {
+    editAvatarPopup.open();
+    editAvatarPopup.setEventListeners();
+  });
 
-    {
-      name: "Lago di Braies",
-      link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg",
-    },
-  ];
+  function toggleLikeButton(id, fillButton, vacateButton) {
+    api.toggleHeartIcon(id, fillButton, vacateButton);
+  }
 
-  const cardsSection = new Section(
-    {
-      items,
-      renderer: (cardData) => {
-        createCard(cardData.name, cardData.link, cardTemplate);
-      },
-    },
-    galleryList
-  );
-  cardsSection.renderItems();
+  // Elements
+  const imageClickHandler = ({ name, link }) => {
+    imagePopup.open({ name, link });
+  };
+  const cardTemplate =
+    document.querySelector("#card-template").content.firstElementChild;
+  const galleryList = document.querySelector("#gallery__list");
 
   const editForm = document.forms["edit-form"];
   const addForm = document.forms["add-form"];
+  const avatarForm = document.forms["avatar-form"];
+  console.log(avatarForm);
 
   const configuration = {
```

Comment:
From the 8th sprint all constants (`initialCards` and `configuration`) should be in file `constants.js` in folder `utils` to clean up the main file.